styles.css

```diff
@@ -0,0 +1,1247 @@
+}
+
+.footer-icons {
+  display: flex;
+  width: 24rem;
+  justify-content: space-between;
+}
+
+.footer-icons a div:hover {
+  background-color: transparent;
+}
+
+.footer-icons a:first-child div i {
+  background: url("assets/icons-sprite.png") -272px -272px;
+}
+
+.footer-icons a:nth-child(2) div i {
+  background: url("assets/icons-sprite.png") -1024px -1024px;
+}
+
+.footer-icons a:nth-child(3) div i {
+  background: url("assets/icons-sprite.png") -1280px -1280px;
+}
+
+.footer-icons a:nth-child(4) div i {
+  background: url("assets/icons-sprite.png") -416px -416px;
+}
+
+.footer-icons a:nth-child(5) div i {
+  background: url("assets/icons-sprite.png") -672px -672px;
+}
+
+.footer-icons a:nth-child(6) div i {
+  background: url("assets/icons-sprite.png") -816px -816px;
+}
+
+.footer-icons a:last-child div i {
+  background: url("assets/icons-sprite.png") -576px -576px;
+}
+
+.footer-copyright {
+  font-family:
+    "Lato",
+    sans-serif;
+  font-size: 0.8rem;
+  background-color: black;
+  height: 6.5rem;
+  color: rgb(196, 196, 196);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.tnwicon {
+  filter: invert(100);
+  transform: scale(0.55);
+  transform-origin: 80%;
+}
+
+.stackpathicon {
+  transform: scale(0.38);
+  transform-origin: 5%;
+}
+
+/* ---------------------------------- MEDIA QUERIES ------------------------------------- */
+
+@media only screen and (min-width: 1600px) {
+  .cover-story {
+    height: 32rem;
+  }
+
+  .cover-logo {
+    background-size: 25rem;
+    width: 25rem;
+  }
+}
+
+@media only screen and (max-width: 1390px) {
+  .cover-logo {
+    width: 33.5vw;
+    background-size: 33.5vw;
+  }
+}
+
+@media only screen and (max-width: 1200px) {
+  .deals-img-div {
+    width: 22.6vw;
+    height: 17vw;
+  }
+
+  .deals-grid-item {
+    width: 22.6vw;
+  }
+
+  .deals-grid {
+    gap: 1vw;
+  }
+}
+
+@media only screen and (max-width: 1024px) {
+  .cover-logo,
+  .cover-date,
+  .cover-gradient {
+    display: none;
+  }
+
+  .cover {
+    filter: brightness(1);
+  }
+
+  .cover-story {
+    height: 60vh;
+  }
+
+  .cover-content h2 {
+    font-size: 2.4rem;
+    line-height: 2.5rem;
+  }
+
+  .bottom-side-cover h2,
+  .top-side-cover h2 {
+    font-size: 1.3em;
+    line-height: 1.5rem;
+    padding-right: 1rem;
+  }
+
+  .logo-gif {
+    display: flex;
+  }
+
+  .deals-grid {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: 1fr 1fr;
+    gap: 2vw;
+  }
+
+  .deals-grid-item {
+    position: relative;
+    width: 30vw;
+    padding-bottom: 1rem;
+  }
+
+  .deals-img-div {
+    width: 30vw;
+    height: 20vw;
+    background: url("assets/deals-img-1.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  .funding-title {
+    flex-direction: column;
+  }
+
+  .funding-grid {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .vanish {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
Your are using desktop first approach for media queries. Because of that you are missing the small breakpoint usually is on  @media only screen and `(max-width: 576px)`

Reply:
Ok Alexandru, I'll fix that. Just now I learned that mobile-first approach is preferable. For the next projects I'll follow this guideline.

Reply:
Actually the project should be mobile-first (base styles for mobile, min-width 768px for tablets and min-width 1024px for desktop). But, it's to late to go back.